src/plugins/intel_cpu/src/nodes/conv.cpp

```diff
@@ -502,8 +548,7 @@ void Convolution::getSupportedDescriptors() {
             createDescriptor({ in_candidate }, { out_candidate });
 
             if ((inputDataType != memory::data_type::bf16 && isNspcAvailable()) ||
-                    (one_of(inputDataType, memory::data_type::bf16) &&
-                    impl::cpu::x64::mayiuse(impl::cpu::x64::avx512_core_amx))) {
+                    (acceptedFormat && impl::cpu::x64::mayiuse(impl::cpu::x64::avx512_core))) {
```

Comment:
`acceptedFormat && impl::cpu::x64::mayiuse(impl::cpu::x64::avx512_core)` seems to be redundant?

Reply:
Removed.